packages/vscode-extension/src/suitecloud.ts

```diff
@@ -32,6 +40,8 @@ export async function activate(context: vscode.ExtensionContext) {
 	context.subscriptions.push(
 		register('suitecloud.adddependencies', new AddDependencies()),
 		register('suitecloud.deploy', new Deploy()),
+		register('suitecloud.importfile', new ImportFiles()),
+		register('suitecloud.importfiles', new ImportFiles()),
```

Comment:
Is it intentional to use the same `new ImportFiles()` for both commands?

Reply:
Yes, this was the refactorization I have made yesterday. There was code duplicated, because logic is the same.
The only thing is that for importfiles, we have a selection of the folders and the files, while in importfile we have already selected the single file to upload

Reply:
Thanks for removing the obsolete import of ImportFiles.